common/src/main/java/com/microsoft/identity/common/internal/util/StringUtil.java

```diff
@@ -230,6 +230,6 @@ public static boolean equalsIgnoreCase(@Nullable final String one, @Nullable fin
      * @return
      */
     public static boolean equalsIgnoreCaseTrim(@Nullable final String one, @Nullable final String two) {
-        return one == two || equalsIgnoreCase(one, two.trim());
+        return one == two || (two != null && equalsIgnoreCase(one, two.trim()));
```

Comment:
Change to StringUtils made after consulting with @tanmaymanolkar1 (thanks for the fix!)

Previously a null in parameter two would cause an exception to be thrown when two.trim() was called.